src/pages/workspaces/List.js

```diff
@@ -282,15 +273,61 @@ export const WorkspaceList = () => {
                   side: 'left'
                 }, [
                   Utils.switchCase(lastRunStatus,
-                    ['success', () => icon('success-standard', { size: 20, style: { color: colors.success() } })],
-                    ['failure', () => icon('error-standard', { size: 20, style: { color: colors.danger(0.85) } })],
-                    ['running', () => icon('sync', { size: 20, style: { color: colors.success() } })]
+                    ['success', () => icon('success-standard', {
+                      size: 20, style: { color: colors.success() },
+                      'aria-label': 'Workflow Status Success'
+                    })],
+                    ['failure', () => icon('error-standard', {
+                      size: 20, style: { color: colors.danger(0.85) },
+                      'aria-label': 'Workflow Status Failure'
+                    })],
+                    ['running', () => icon('sync', {
+                      size: 20, style: { color: colors.success() },
+                      'aria-label': 'Workflow Status Running'
+                    })]
                   )
                 ])
               ])
             ])
           },
           size: { basis: 30, grow: 0, shrink: 0 }
+        }, {
+          headerRenderer: () => div({ className: 'sr-only' }),
```

Comment:
You need to add the name of what the column is rendering (for the screen reader). Right now it reads "Blank". See line 308 below.